spec/models/communication_spec.rb

```diff
@@ -85,6 +85,19 @@
     end
   end
 
+  describe ".create_with_imo_details" do
+    it "creates a communication" do
+      patient = create(:patient)
+      appt = create(:appointment, patient: patient)
+      notification = create(:notification, subject: appt, patient: patient)
+      expect {
+        Communication.create_with_imo_details!(appointment: notification.subject,
+                                               notification: notification,
+                                               result: nil)
+      }.to change { Communication.count }.by(1)
```

Comment:
Can we add more specific assertions to this spec around the details of the `Communication` record we just created?